src/contexts/Filter-context.js

```diff
@@ -0,0 +1,25 @@
+import { createContext, useReducer } from "react";
+import { filterReducer } from "../reducer/filterReducer";
+const FilterContext = createContext();
+
+var initialFilter;
```

Comment:
Why you're using **` var`** here.

Reply:
because I am passing this object to my reducer function in some other file